gradle/versions.gradle

```diff
@@ -17,10 +17,14 @@ dependencyManagement {
   dependencies {
     dependency 'com.fasterxml.jackson.core:jackson-databind:2.10.1'
     dependency 'com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.10.1'
+
     dependency 'com.github.tomakehurst:wiremock-jre8:2.25.1'
 
     dependency 'com.google.auto.service:auto-service:1.0-rc6'
 
+    dependency 'com.google.dagger:dagger:2.27'
```

Comment:
We don't need the annotationProcessor ? com.google.dagger:dagger-compiler

Reply:
It's the next line. This file just sets library versions.

Reply:
Ok.